doc_src/cmds/bind.rst

```diff
@@ -179,7 +179,9 @@ The following special input functions are available:
     move one whitespace-delimited word to the right
 
 ``forward-char``
-    move one character to the right
+    move one character to the right; or in a menu select the item to the right;
+    or if it at the end of the input and an autosuggestion is available accept
+    it completely
```

Comment:
Thanks, this looks nice.

I'm not a fan of the "completely" wording though.
I think our only menu is the completion pager, but we rarely use the word menu (only twice in doc_src/interactive.rst) which is an argument for using "(completion) pager".
While at it, we can also document the same behavior in backward-char

so how about something like

```
diff --git a/doc_src/cmds/bind.rst b/doc_src/cmds/bind.rst
index 9de9ed5ca..b6f45f429 100644
--- a/doc_src/cmds/bind.rst
+++ b/doc_src/cmds/bind.rst
@@ -91,6 +91,7 @@ The following special input functions are available:
 ``accept-autosuggestion``
-    accept the current autosuggestion completely
+    accept the current autosuggestion
 
 ``backward-char``
-    moves one character to the left
+    move one character to the left.
+    If the completion pager is active, select the previous completion instead.
 
@@ -181,6 +182,7 @@ The following special input functions are available:
 ``forward-char``
-    move one character to the right
+    move one character to the right; or if at the end of the commandline, accept the current autosuggestion.
+    If the completion pager is active, select the next completion instead.
 
 ``forward-single-char``
-    move one character to the right; if an autosuggestion is available, only take a single char from it
+    move one character to the right; or if at the end of the commandline, accept a single char from the current autosuggestion.
 
```

Reply:
OK, will do - unless you prefer to just merge that yourself?

> -    move one character to the right; if an autosuggestion is available, only take a single char from it

I'd drop "only" from that clause too.

Reply:
> OK, will do

that'd be nice, it's always helpful to double check these

> I'd drop "only" from that clause too. 

sounds good